toc.yml

```
@@ -1,6 +1,6 @@
 - name: Docs
   href: docs/
 - name: OpenAPI
-  href: /docs/openapi/swagger-ui.html
+  href: api/docs/openapi/swagger-ui.html
```

hm, I think this would break the local builds as those are served from the root path (eg. `http://localhost:5000/` and not `http://localhost:5000/api`)

I think a better solution would be to simply change the absolute path to a relative path (as you did, without the `api/` prefix:
```suggestion
  href: docs/openapi/swagger-ui.html
```

---

I completely missed that it was the leading slash that could cause it. 

I set up a local build but was unable to access `http://localhost:5000/docs/openapi/swagger-ui.html` so I'm not really sure how I can verify, but I'll commit the suggestion for now since that makes more sense to me.